core/storage/leveldb/leveldb.cpp

```diff
@@ -77,21 +77,32 @@ namespace kagome::storage {
     std::string value;
     auto status = db_->Get(ro_, make_slice(key), &value);
     if (status.ok()) {
-      // FIXME: is it possible to avoid copying string -> Buffer?
+      // cannot move string content to a buffer
       return Buffer{}.put(value);
     }
 
-    // not always an actual error so don't log it
+    return status_as_error(status, logger_);
+  }
+
+  outcome::result<boost::optional<Buffer>> LevelDB::tryGet(
```

Comment:
I don't think that this change is significant. I think, it doesn't add any additional functionality. It just replaces NO_VALUE error with boost::none. Is that worth it? I think it is even erroneous, because technically boost::none would mean that key existed, but it is not!

Reply:
I disagree that boost::none would mean that a key existed, it is a common way to indicate that a container does not contain the requested value. It would be pretty easy to forget to check for NO_VALUE in common get() method, while it typically should not be propagated like any other error, but processed in the calling function as it is an expected behaviour of getting a key from a storage.